api_volontaria/settings.py

```diff
@@ -173,11 +173,17 @@
 USE_TZ = True
 
 
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/2.0/howto/static-files/
+# Force local storage for unittests. Temporary.
+# if len(sys.argv) > 1 and sys.argv[1] == 'test':
+#     STATIC_URL = '/static/'
+#     STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
+# else:
+#     STATIC_URL = config('STATIC_URL', default='/static/')
+#     STATICFILES_STORAGE = config('STATICFILES_STORAGE',
+#                                  default='django.contrib.staticfiles.storage.StaticFilesStorage')
+# if STATICFILES_STORAGE == 'django.contrib.staticfiles.storage.StaticFilesStorage':
+#     STATIC_ROOT = 'static/'
```

Comment:
Code should not be commented, if needed you can remove it
PS: The old comment was cool to have in the settings in think